tests/conftest.py

```diff
@@ -0,0 +1,53 @@
+import json
+import random
+import string
+
+import pytest
+import requests
+from jinja2 import Template
+
+_AUTH_URL = 'https://urs.earthdata.nasa.gov/oauth/authorize?response_type=code' \
+            '&client_id=BO_n7nTIlMljdvU6kRRB3g' \
+            '&redirect_uri=https://auth.asf.alaska.edu/login'
+
+
+class HelperFunctions:
+    @staticmethod
+    def create_jobs_post(template):
```

Comment:
This name suggests that the function does a POST request, perhaps `get_jobs_json()` or `get_jobs_payload()`

Reply:
Yeah, I wasn't happy with the name either --  I like `get_jobs_payload` better